docs/source/dev/supported_vrplib_fields.rst

```diff
@@ -26,6 +26,8 @@ PyVRP supports the following specifications:
    ``CAPACITY``
       Homogeneous capacity of all vehicles in the instance.
       The number of vehicles is provided using the :term:`VEHICLES` specification.
+      To specify heterogeneous capacities, see :term:`CAPACITY_SECTION`.
+      When :term:`CAPACITY` and :term:`CAPACITY_SECTION` are both provided, the data from :term:`CAPACITY_SECTION` is used.
```

Comment:
I think we need to do this like with `SERVICE_TIME_SECTION` - just describe the section, and leave it implicit that a specification is also OK (and assumed to then hold for all clients/vehicles/as appropriate).

I am not sure what we should do if both a specification and a section is given. That is not unambiguous. I think that should be an error (possibly already in VRPLIB; because it's overwriting an existing field).

Reply:
- I've included `SERVICE_TIME` as a specification. I think it's beter to be implicit about it.
- When a field allows both specification and section, the specification refers to the section.
- We no longer talk about instances that include both specifications and sections for a single field - this will be handled by `vrplib`.